examples/website/scatterplot/app.jsx

```diff
@@ -32,6 +32,13 @@ export default function App({
     new ScatterplotLayer({
       id: 'scatter-plot',
       data,
+      pickable: true,
+      onHover: info => {
+        if (info.object) {
+          console.log(info.object);
+        }
+      },
```

Comment:
It might be worth showing when it is unhovered (null) as well as that is an important part of the functionality
```suggestion
      onHover: info => console.log(info?.object);
```